FileShareClient/FileShareApiClient.cs

```diff
@@ -212,6 +213,19 @@ public async Task<IResult<BatchAttributesSearchResponse>> BatchAttributeSearch(s
             return result;
         }
 
+        public async Task<Stream> DownloadZipFileAsync(string batchId, CancellationToken cancellationToken)
```

Comment:
Please change `DownloadZipFileAsync` to return `Task<IResult<Stream>>`

The IResult interface is intended to return additional error information to callers without the callers having to couple themselves to the format/content/type of exceptions returned by the API